packages/components/src/composite/legacy/test/index.tsx

```diff
@@ -275,7 +275,7 @@ describe.each( [
 		const { item2 } = getOneDimensionalItems();
 
 		await press.Tab();
-		expect( item2 ).toHaveFocus();
+		await waitFor( () => expect( item2 ).toHaveFocus() );
```

Comment:
Perhaps some `sleep()` before the `Tab()` (as suggested [here](https://github.com/WordPress/gutenberg/pull/58629#issuecomment-1924875249) would be a good improvement too?

Reply:
Yes, that might help. The Tabs test was fine, but I could still get this Composite one to flake.

Reply:
Done in 1eb47f204b335087da115bea2a4272c827d65b40